### PR TITLE
Quiz page

### PR DESCRIPTION
**Pull request by PuneetP16**
features added: 
- rules page
- timers for a quiz question
- quiz question page
- score board display on quiz complete page
- login with google and facebook

### CODE REVIEW

File: src/components/Cards/QuestionCard/QuestionCard.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
+:root {
+	--COLOR-BS-OPTION: #065d8814;
+	--COLOR-CORRECT: #7fffd4;
+	--COLOR-WRONG: #453326;
+	--COLOR-BG-WRONG: #ed8181;
+}
+
+.question__card {
+	color: var(--COLOR-TXT);
```
Comment (dev-ashishsethi) [inline src/components/Cards/QuestionCard/QuestionCard.css:9]: fallback value for colors should be given

File: src/components/Cards/QuestionCard/QuestionCard.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+	font-weight: 400;
+	padding: 3rem 2rem;
+	width: 100%;
+	box-shadow: 0 4px 12px var(--COLOR-BS-OPTION);
+	cursor: pointer;
+}
+
+.option__container:hover {
+	/* border: 1px solid; */
+	background-color: rgba(0, 0, 0, 0.1);
```
Comment (dev-ashishsethi) [inline src/components/Cards/QuestionCard/QuestionCard.css:98]: use varaible for colors

File: src/components/Cards/QuestionCard/QuestionCard.jsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+						if (questionIndex < totalQuestions - 1) {
+							setQuestionIndex((prevIndex) => prevIndex + 1);
+						}
+						setTimer(5);
+					}
+					return () => clearTimeout(timeoutId);
+				}
+			}
+		}
+	}, [timer, questionIndex, modal]);
```
Comment (dev-ashishsethi) [inline src/components/Cards/QuestionCard/QuestionCard.jsx:92]: looks like this could have been simplified

**Review by dev-ashishsethi — APPROVED** [review]
do check comments

File: src/components/Cards/QuestionCard/QuestionCard.css (hunk 1 above)
Comment (PuneetP16) [reply src/components/Cards/QuestionCard/QuestionCard.css:9]: Will look into it

File: src/components/Cards/QuestionCard/QuestionCard.css (hunk 2 above)
Comment (PuneetP16) [reply src/components/Cards/QuestionCard/QuestionCard.css:98]: Updated it.

File: src/pages/Authentication/Login/Login.jsx [hunk 4]
```diff
@@ -127,7 +141,26 @@ export const Login = () => {
...
+						{bxIcons.google}
+						<span>Login with Google</span>
+					</button>
+					<button
+						onClick={facebookSigninHandler}
+						className="form__login_btn form__google btn btn--outline--primary"
+					>
+						{bxIcons.facebook}
+						<span>Login with Facebook</span>
+					</button>
```
Comment (m26unkwn) [inline src/pages/Authentication/Login/Login.jsx:163]: You can extract it in array of objects than you can map over it.

**Review by m26unkwn — APPROVED** [review]
Remove unwanted console.log from your code 
and than you are good to go.

File: src/pages/Authentication/Login/Login.jsx (hunk 4 above)
Comment (PuneetP16) [reply src/pages/Authentication/Login/Login.jsx:163]: Ok. would look into it.

**Pull request merged**